### DISCUSSION TITLE
Which layer of SCT for FindMarkers

### DISCUSSION THREAD

**Discussion by viktorfeketa** [satijalab/seurat#8669] [Q&A]
There has been a lot of discussions and confusion about this in the past, so I just wanted to check what is the latest thinking by the community and/or Seurat team.
For an integrated dataset, normalized by SCT at all steps, which layer of the SCT assay should be used to identify cluster markers (using FindMarkers and FindConservedMarkers) and for DE testing between different conditions (using FindMarkers)? Or should the "RNA" assay be used instead?

**Comment by wudustan** [comment]
Looking through the SCTransform vignette https://satijalab.org/seurat/articles/sctransform_vignette

>To assist with visualization and interpretation, we also convert Pearson residuals back to ‘corrected’ UMI counts. You can interpret these as the UMI counts we would expect to observe if all cells were sequenced to the same depth. If you want to see exactly how we do this, please look at the correct function [here](https://github.com/ChristophH/sctransform/blob/master/R/denoise.R).
>The ‘corrected’ UMI counts are stored in pbmc[["SCT"]]$counts. We store log-normalized versions of these corrected counts in pbmc[["SCT"]]$data, which are very helpful for visualization.

The answer would appear to be to use `seurat[["SCT"]]$data` unless you're using "negbinom", "poisson", or "DESeq2" methods in which case `seurat[["SCT"]]$counts`
